tests/toranj/test-002-form.py

```diff
@@ -53,7 +53,7 @@
 # Test implementation
 
 # default values after reset
-DEFAULT_KEY = '[00112233445566778899AABBCCDDEEFF]'
+DEFAULT_KEY = '[00000000000000000000000000000000]'
```

Comment:
This is fine, but since we may no longer have the notion of a default key in OT, how about we simplify the test script here as well. We can consider  removing the `DEFAULT_KEY` and all checks against it in the test script.

Reply:
Done.